fpga/src/main/scala/arty/TestHarness.scala

```diff
@@ -4,6 +4,8 @@ import chisel3._
 
 import freechips.rocketchip.diplomacy.{LazyModule}
 import freechips.rocketchip.config.{Parameters}
+import sifive.blocks.devices.spi._
+import sifive.blocks.devices.gpio._
 
```

Comment:
Is this necessary?

Reply:
This was unintended. Will quash changes to TestHarness.scala